apps/extension/client/package.json

```
@@ -0,0 +1,96 @@
+{
+  "name": "stately-vscode",
+  "displayName": "XState VSCode",
+  "description": "Visual editing, autocomplete and typegen for XState",
+  "license": "MIT",
+  "author": "Stately Team",
+  "version": "1.6.1",
```

I'm not completely sure how this will behave when publishing to the extension store etc but from the PoV of Changesets+npm it's probable safer to start with this:

```suggestion
  "version": "1.6.1",
  "private": true
```